lib/improver/nbhood/vicinity.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not entirely convinced this belongs in a nbhood sub-directory, but I'm open to persuasion.